package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "",
+  "main": "webpack.config.js",
+  "scripts": {
+    "build": "npx webpack",
+    "start": "npx webpack serve"
+  },
```

Comment:
- Missing a ``README`` file. Implement information about your project, how to use it, how to install it, how to run it, a License and your social media contact.